shoop/admin/modules/products/views/toolbars.py

```diff
@@ -94,33 +96,66 @@ def _get_variation_and_package_menu_items(self, product):
                     icon="fa fa-eye",
                     url=get_model_url(sib),
                 )
-        elif package_parent:
+
+    def _get_package_menu_items(self, product):
+        package_parent = product.is_package_parent()
+        package_child = product.is_package_child()
+        package_url = self._get_package_url(product)
+        if package_parent:
             yield DropdownDivider()
-            yield DropdownHeader(text=_("Variations"))
+            yield DropdownHeader(text=_("Packages"))
             yield DropdownItem(
                 text=_("Manage Package"),
                 icon="fa fa-cube",
-                url="#",  # TODO: Implement manage packages
+                url=package_url,
             )
             for child in product.get_all_package_children():
                 yield DropdownItem(
                     text=_("Child: %s") % child,
                     icon="fa fa-eye",
                     url=get_model_url(child),
                 )
-
-        package_parents = list(product.get_all_package_parents())
-        if package_parents:
+        elif package_child:
             yield DropdownDivider()
-            yield DropdownHeader(text=_("Variations"))
-            for parent in package_parents:
+            yield DropdownHeader(text=_("Packages"))
+            for parent in product.get_all_package_parents():
                 yield DropdownItem(
-                    text=_("Package Parent: %s") % parent,
+                    text=_("Manage Package"),
+                    icon="fa fa-cube",
+                    url=self._get_package_url(parent),
+                )
+                yield DropdownItem(
+                    text=_("Parent: %s") % parent,
                     icon="fa fa-eye",
                     url=get_model_url(parent),
                 )
+                for sib in parent.get_all_package_children():
+                    if sib != product:
+                        yield DropdownItem(
+                            text=_("Subling: %s") % sib,
+                            icon="fa fa-eye",
+                            url=get_model_url(sib),
+                        )
 
-        if not (variation_parent or variation_child or package_parent):
+    def _get_variation_and_package_menu_items(self, product):
+        package_parent = product.is_package_parent()
+        package_child = product.is_package_child()
+        variation_parent = product.is_variation_parent()
+        variation_child = product.is_variation_child()
+        package_url = self._get_package_url(product)
+        variation_url = reverse("shoop_admin:product.edit_variation", kwargs={"pk": product.pk})
+        for item in self._get_variation_menu_items(product):
```

Comment:
Ordering here is little bit off I think. No need to get variation or parent items if the product is ``not (variation_parent or variation_child or package_parent or package_child)``